Sources/SwiftUI/SwiftUI.swift

```diff
@@ -0,0 +1,3 @@
+// Copyright 2025 Skip
+// SPDX-License-Identifier: LGPL-3.0-only WITH LGPL-3.0-linking-exception
+@_exported import SkipFuseUI
```

Comment:
Given that this is used for Android only, it could export SkipSwiftUI directly, right? 

Do we want to continue to maintain "SkipFuseUI", "SkipSwiftUI", and "SkipUI" targets? I guess we have no choice for backwards compatibility. We should probably comment somewhere on why we have them